test/framework/module_generator.py

```diff
@@ -50,10 +50,10 @@
 
 
 class ModuleGeneratorTest(EnhancedTestCase):
-    """ testcase for ModuleGenerator """
+    """ testcase for ModuleGeneratorTcl """
```

Comment:
that seems ideal in the long term.
for now, generating the text and veryfing it's as expected is enough imho